<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createToApiPhp</class>
 <widget class="QMainWindow" name="createToApiPhp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleMail">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlePass">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="confirmTitlePass">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Pass :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signupBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textMail">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textPass">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textPassConfirm">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errorZone">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
